earth2studio/models/px/aifsens.py

```diff
@@ -0,0 +1,832 @@
+
+import numpy as np
+import torch
+
+from earth2studio.models.auto import AutoModelMixin, Package
+from earth2studio.models.batch import batch_coords, batch_func
+from earth2studio.models.px.base import PrognosticModel
+from earth2studio.models.px.utils import PrognosticMixin
+from earth2studio.utils import handshake_coords, handshake_dim
+from earth2studio.utils.imports import (
+    OptionalDependencyFailure,
+    check_optional_dependencies,
+)
+from earth2studio.utils.type import CoordSystem
+
+try:
+    import anemoi.models  # noqa: F401
+    import earthkit.regrid  # noqa: F401
+    import ecmwf.opendata  # noqa: F401
+    import flash_attn  # noqa: F401
+except ImportError:
+    OptionalDependencyFailure("aifsens")
+
+VARIABLES = [
+    "q50",
+    "q100",
+    "q150",
+    "q200",
+    "q250",
+    "q300",
+    "q400",
+    "q500",
+    "q600",
+    "q700",
+    "q850",
+    "q925",
+    "q1000",
+    "t50",
+    "t100",
+    "t150",
+    "t200",
+    "t250",
+    "t300",
+    "t400",
+    "t500",
+    "t600",
+    "t700",
+    "t850",
+    "t925",
+    "t1000",
+    "u50",
+    "u100",
+    "u150",
+    "u200",
+    "u250",
+    "u300",
+    "u400",
+    "u500",
+    "u600",
+    "u700",
+    "u850",
+    "u925",
+    "u1000",
+    "v50",
+    "v100",
+    "v150",
+    "v200",
+    "v250",
+    "v300",
+    "v400",
+    "v500",
+    "v600",
+    "v700",
+    "v850",
+    "v925",
+    "v1000",
+    "w50",
+    "w100",
+    "w150",
+    "w200",
+    "w250",
+    "w300",
+    "w400",
+    "w500",
+    "w600",
+    "w700",
+    "w850",
+    "w925",
+    "w1000",
+    "z50",
+    "z100",
+    "z150",
+    "z200",
+    "z250",
+    "z300",
+    "z400",
+    "z500",
+    "z600",
+    "z700",
+    "z850",
+    "z925",
+    "z1000",
+    "u10m",
+    "v10m",
+    "d2m",
+    "t2m",
+    "lsm",
+    "msl",
+    "sdor",
+    "skt",
+    "slor",
+    "sp",
+    "tcw",
+    "z",
+    "cp06",
+    "tp06",
+    "cos_latitude",
+    "cos_longitude",
+    "sin_latitude",
+    "sin_longitude",
+    "cos_julian_day",
+    "cos_local_time",
+    "sin_julian_day",
+    "sin_local_time",
+    "insolation",
+    "stl1",
+    "stl2",
+    "ssrd",
+    "strd",
+    "sf",
+    "tcc",
+    "mcc",
+    "hcc",
+    "lcc",
+    "u100m",
+    "v100m",
+    "ro",
+]  # from config.json >> dataset.variables
+
+
+@check_optional_dependencies()
+class AIFSENS(torch.nn.Module, AutoModelMixin, PrognosticMixin):
+    """Artificial Intelligence Forecasting System Ensemble (AIFS ENS v1.0), a
+    probabilistic, ensemble-based forecast model from the European Centre for
+    Medium-Range Weather Forecasts (ECMWF). AIFS ENS uses a GNN encoder/decoder with a
+    sliding-window transformer processor, trained on ERA5 reanalysis and operational NWP
+    analyses, and is run four times daily with a 6-hour time step. The model is trained
+    with a CRPS objective over a small ensemble to provide calibrated probabilistic
+    output.
+
```

Comment:
Missing the arguments in the doc string

Reply:
Oh dang, was missing from AIFS as well...